### DISCUSSION TITLE
Google Data Studio integration

### DISCUSSION THREAD

**Discussion by metmarkosaric** [plausible/analytics#828] [Anything Else]
More details on Google Data Studio and connectors: https://developers.google.com/datastudio/connector/get-started

(request via email)

**Comment by metmarkosaric** [comment]
i was just informed that there now is a Plausible connector for Google Data Studio. It is not made by us and it is a paid product, but for those that require it, it could be worth checking out https://www.epilocal.com/products/plausible-data-studio-connector/

**Comment by Hricha-Shandily** [comment]
The official Plausible Analytics Looker Studio Connector is now live! All details available in our setup guide: https://plausible.io/docs/looker-studio
